modules/mobile/app/models/mobile/v0/adapters/lighthouse_individual_claims.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Adapters
+      class LighthouseIndividualClaims
+        PHASE_TYPE_TO_NUMBER = {
+          CLAIM_RECEIVED: 1,
+          UNDER_REVIEW: 2,
+          GATHERING_OF_EVIDENCE: 3,
+          REVIEW_OF_EVIDENCE: 4,
+          PREPARATION_FOR_DECISION: 5,
+          PENDING_DECISION_APPROVAL: 6,
+          PREPARATION_FOR_NOTIFICATION: 7,
+          COMPLETE: 8
+        }.freeze
+
+        # rubocop:disable Metrics/MethodLength
+        def parse(claim)
+          attributes = claim.dig('data', 'attributes')
+          phase_change_date = attributes.dig('claimPhaseDates', 'phaseChangeDate')
+          events_timeline = events_timeline(attributes)
+
+          Mobile::V0::Claim.new(
+            {
+              id: claim['data']['id'],
+              date_filed: attributes['claimDate'],
+              min_est_date: attributes['minEstClaimDate'],
+              max_est_date: attributes['maxEstClaimDate'],
+              phase_change_date:,
+              open: attributes['closeDate'].nil?,
+              waiver_submitted: attributes['evidenceWaiverSubmitted5103'],
+              documents_needed: attributes['documentsNeeded'],
+              development_letter_sent: attributes['developmentLetterSent'],
+              decision_letter_sent: attributes['decisionLetterSent'],
+              phase: phase(attributes.dig('claimPhaseDates', 'latestPhaseType')),
+              ever_phase_back: nil,
+              current_phase_back: attributes.dig('claimPhaseDates', 'currentPhaseBack'),
+              requested_decision: attributes['evidenceWaiverSubmitted5103'],
+              claim_type: attributes['claimType'],
+              contention_list: attributes['contentionList'],
+              va_representative: nil,
```

Comment:
va_representative also doesn't have a replacement but is used by the app. Product is looking into an alternative: https://dsva.slack.com/archives/C04KHCT3ZMY/p1683734273336449